app/views/layouts/application.html.erb

```diff
@@ -35,7 +35,7 @@
     <%= yield :head %>
   </head>
 
-  <body data-controller="preserve-scroll">
+  <body class="<%= hotwire_native_app? ? "hotwire-native" : "desktop" %>" data-controller="preserve-scroll">
```

Comment:
What if we put a simple `data-hotwire-app` attribute on the `<html>` element itself? Then it would be on the same element as before

Reply:
Sure, sounds good to me! I usually try to use primitive selectors like `.desktop` instead of `:not([data-hotwire-app])` when possible, but I see the benefits of leaving it on the `<html>` element. I'll update the PR.